src/test/java/catan/controllers/testcases/play/BuildRoadTest.java

```diff
@@ -145,6 +199,7 @@ public void should_successfully_build_road_on_empty_edge_if_has_neighbour_road_t
 
     }
 
+    @Ignore
```

Comment:
all of test cases in BuildRoadTest was ignored before by annotation of whole class. I've just put Ignore annotation to each test case separately.